src/mmw/js/src/compare/controllers.js

```diff
@@ -16,28 +18,87 @@ var CompareController = {
             App.currProject
                 .fetch()
                 .done(function() {
+                    setupProjectCopy();
```

Comment:
This idea would address point 1 of the comment in setupProjectCopy 
 -changes to the results and inputs are not saved to the server

But would not address points 2 and 3
-changes to the results and inputs are not present after hitting the back button (and project is unsaved)
-the original project (without changes) is saved when logging in

Originally, I was setting the user_id of the copied project to 0, so that the logic for shared projects would be used, but I decided to add an attempt_save flag to the ProjectModel and ScenarioModel to be more explicit about not attempting to save.